Secure-Software-Development-Policy/Manifesto.md

```diff
@@ -0,0 +1,14 @@
+# Zowe Secure Software Development Manifesto
+
+- We strive to develop the most secure modern tools and interfaces for access to mainframe resources and services.
+
+- Zowe will never compromise on mainframe security standards and in some cases would provide additional measures above that level, while opening the mainframe for integrations with the distributed world.
+
+- Ze strive to never introduce a feature or change that circumvents or allows circumvention of z/OS security infrastructure.
+
+- We commit to secure development best practices.
```

Comment:
Is it possible to commit to something so vague? Can we "Strive to follow secure development best practices as they evolve over time"